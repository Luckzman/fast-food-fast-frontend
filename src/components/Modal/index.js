import React from 'react';
import { Button, Modal } from 'semantic-ui-react';
import FormTab from '../FormTab';

const ModalContainer = () => (
  <Modal trigger={<Button>Login</Button>} closeIcon size="mini">
    <Modal.Content>
      <FormTab />
    </Modal.Content>
  </Modal>
);

export default ModalContainer;
