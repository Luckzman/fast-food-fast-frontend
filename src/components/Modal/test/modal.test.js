import React from 'react';
import { shallow } from 'enzyme';
import Modal from '..';

describe('App', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Modal />);
    expect(wrapper).toMatchSnapshot();
  });
});
