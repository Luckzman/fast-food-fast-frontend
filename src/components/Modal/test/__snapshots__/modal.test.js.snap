// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
<Modal
  centered={true}
  closeIcon={true}
  closeOnDimmerClick={true}
  closeOnDocumentClick={false}
  dimmer={true}
  eventPool="Modal"
  size="mini"
  trigger={
    <Button
      as="button"
    >
      Login
    </Button>
  }
>
  <ModalContent>
    <FormTab />
  </ModalContent>
</Modal>
`;
